README.md

```diff
@@ -15,12 +15,17 @@ If you don't have YunoHost, please consult [the guide](https://yunohost.org/#/in
 
 ## Overview
 
-Terminal in browser over HTTP/HTTPS. (Ajaxterm/Anyterm alternative)
+Terminal over HTTP and HTTPS. WeTTy is an alternative to ajaxterm and anyterm but much better than them because WeTTy uses xterm.js which is a full fledged implementation of terminal emulation written entirely in JavaScript. WeTTy uses websockets rather then Ajax and hence better response time.
```

Comment:
Why this change ?

Reply:
I mean the current "overview" is already enough and this just adds "noise".

Reply:
`Why this change ?` 
why not... :) We now have a Description.md available for more in depth description (instead of using the description from the manifest.)